Public/Src/FrontEnd/Script/Ast/Expressions/SwitchExpression.cs

```diff
@@ -90,16 +90,25 @@ protected override EvaluationResult DoEval(Context context, ModuleLiteral env, E
 
             foreach (var clause in Clauses)
             {
-                var match = clause.Match.Eval(context, env, frame);
-                if (match.IsErrorValue)
+                if (clause.IsDefaultFallthrough)
                 {
-                    return match;
+                    return clause.Expression.Eval(context, env, frame);
                 }
-
-                if (expression.Equals(match))
+                else
                 {
-                    return clause.Expression.Eval(context, env, frame);
+                    var match = clause.Match.Eval(context, env, frame);
+                    if (match.IsErrorValue)
+                    {
+                        return match;
+                    }
+
+                    if (expression.Equals(match))
+                    {
+                        return clause.Expression.Eval(context, env, frame);
+                    }
                 }
+
+                

```

Comment:
Remove

Reply:
I miss our style checks...

---
In reply to: [274196949](https://github.com/Microsoft/BuildXL/pull/123#discussion_r274196949) [](ancestors = 274196949)